build.rs

```diff
@@ -0,0 +1,22 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // We need to build the proto files so that we can use them in the tests
+    println!("cargo:rerun-if-changed=crates/kubelet/proto/deviceplugin/v1beta1/deviceplugin.proto");
+
+    let builder = tonic_build::configure()
+        .format(true)
+        .build_client(true)
+        .build_server(true);
+
+    println!("cargo:warning=Build file ran");
```

Comment:
Left over debug I think from when we were pairing

Reply:
Nice catch! ec47ee860d808c84f10a4a64381a399688f356b4